src/Components/Render.cpp

```diff
@@ -0,0 +1,30 @@
+#include "Render.h"
+#include "MyGLWidget.h"
+
+Render::Render(MyGLWidget* _glWidget)
+{
+	glWidget = _glWidget;
+}
+
+void Render::start() {
+	srand(time(NULL));
```

Comment:
srand() is usually called in main(), otherwise, we'll be seeding it with each Render instance.
Consider using the random library in boost for better randomness. (Longer cycle)